spk/radarr/src/service-setup.sh

```diff
@@ -55,39 +41,17 @@ service_preupgrade ()
         # Create, in case it's missing for some reason
         mkdir ${CONFIG_DIR} >> ${INST_LOG} 2>&1
     fi
-
-    # Is Installed Radarr Binary Ver. >= SPK Radarr Binary Ver.?
-    CUR_VER=$(${MONO_PATH}/monodis --assembly ${RADARR} | grep "Version:" | awk '{print $2}')
-    echo "Installed Radarr Binary: ${CUR_VER}" >> ${INST_LOG}
-    SPK_VER=$(${MONO_PATH}/monodis --assembly ${SPK_RADARR} | grep "Version:" | awk '{print $2}')
-    echo "Requested Radarr Binary: ${SPK_VER}" >> ${INST_LOG}
-    if [ "${CUR_VER//.}" -ge "${SPK_VER//.}" ]; then
-        echo 'KEEP_CUR="yes"' > ${CONFIG_DIR}/KEEP_VAR
-        echo "[KEEPING] Installed Radarr Binary - Upgrading Package Only" >> ${INST_LOG}
-        mv ${SYNOPKG_PKGDEST}/share ${INST_VAR}
-    else
-        echo 'KEEP_CUR="no"' > ${CONFIG_DIR}/KEEP_VAR
-        echo "[REPLACING] Installed Radarr Binary" >> ${INST_LOG}
-    fi
 }
 
 service_postupgrade ()
 {
-    # Restore Current Radarr Binary if Current Ver. >= SPK Ver.
-    . ${CONFIG_DIR}/KEEP_VAR
-    if [ "$KEEP_CUR" == "yes" ]; then
-        echo "Restoring Radarr version from before upgrade" >> ${INST_LOG}
-        rm -fr ${SYNOPKG_PKGDEST}/share >> ${INST_LOG} 2>&1
-        mv ${INST_VAR}/share ${SYNOPKG_PKGDEST}/ >> ${INST_LOG} 2>&1
-        set_unix_permissions "${SYNOPKG_PKGDEST}/share"
-    fi
+    # Make Radarr do an update check on start to avoid possible Radarr
+    # downgrade when synocommunity package is updated
+    touch ${CONFIG_DIR}/update_required
```

Comment:
creating this file should prompt radarr to do an update check on startup, negating the need for manually preserving the binaries